### ISSUE TITLE
Error exporting text with non-ascii characters

### ISSUE THREAD

**Issue by GoogleCodeExporter** [halamalala/inkscape2tikz#12]
```
What steps will reproduce the problem?

 1. Create a document with containing some with non-ascii characters, like
the spanish accents
 2. Try to export.

What is the expected output? What do you see instead?

I see a dialog with this backtrace:

Traceback (most recent call last):
  File "/home/jesus/.inkscape/extensions/tikz_export.py", line 985, in <module>
    main_inkscape()
  File "/home/jesus/.inkscape/extensions/tikz_export.py", line 975, in
main_inkscape
    effect.affect()
  File "/home/jesus/.inkscape/extensions/inkex.py", line 155, in affect
    self.output()
  File "/home/jesus/.inkscape/extensions/tikz_export.py", line 945, in output
    f.write(self.output_code)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xc1' in
position 202: ordinal not in range(128)

What version of the product are you using? On what operating system?

The lastest, from repository.

Please provide any additional information below.
Obviously is the classical problem in Python with byte string, unicode
strings and encodings.


```

Original issue reported on code.google.com by `aplatan...@gulic.org` on 24 Aug 2009 at 9:55

**Comment by GoogleCodeExporter** [comment]
```
The easiest solution to this problem is probably to output unicode (utf8) and 
rely on
\usepackage[utf8]{inputenc} on the TeX-side.  
```

Original comment by `kjel...@gmail.com` on 25 Aug 2009 at 6:34
- Changed state: **Accepted**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 22ae53c652.
```

Original comment by `kjel...@gmail.com` on 26 Aug 2009 at 4:54
- Changed state: **Fixed**
